ml-agents/README.md

```diff
@@ -1,5 +1,22 @@
-# Python API
+# Unity ml-agents interface and trainers
 
+The `mlagents` package contains two components : The low level API which allows you to interact directly with a Unity Environment and a training component whcih allows you to train agents in Unity Environments using our implementations of reinforcement learning or imitation learning.
+
+## Installation
+
+The `ml-agents` package can be installed using:
+
+```
+pip install mlagents
+```
+
+or by running the following from the `ml-agents` directory of the repository:
+
+```
+pip install .
+```
+
+## `mlagents.envs`
```

Comment:
Markdown likes it better when there is a single blank line above and below a heading. I think there should be a blank line below this heading.